### DISCUSSION TITLE
1.2.0 Nightly builds

### DISCUSSION THREAD

**Discussion by pedjas** [rustdesk/rustdesk#3310] [General]
1.2 version is way improved and way more functional compared to actual 1.1.9. 

https://github.com/rustdesk/rustdesk/releases/tag/nightly

There is issue with versioning. All nightly builds are released as version 1.2.0 so if 1.2.0 is already installed, new build, when started, does not recognize that it is actually newer version and does not offer to update already installed one.

Is is possible to set new version number for each build?

Are there plans to release official 1.2 version in near future?

**Comment by elvirdz** [comment]
rustdesk.exe --reinstall or rustdesk.exe --update
or you make shortcut of new rustdesk.exe and set  --update in source

example:
"C:\Downloads\rustdesk.exe" --update

**Reply by pedjas** [reply]
That worked. Thanks.
